### PR TITLE
FX-3129 feat: Added /fair-organizer/:slug route

### PR DESCRIPTION
**Pull request by anastasiapyzhik**
[FX-3129]

Screenshot 2021-07-27 at 09 53 08

[FX-3129]: https://artsyproduct.atlassian.net/browse/FX-3129

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Routes added to `routes.tsx` should have a corresponding cypress.js smoke test. See the `cypress/integration` folder for examples.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 01c400b241f92a8fb2382c2ce0c62818eb9e52e7
</p>

**Comment by dblandin** [comment]
> Routes added to routes.tsx should have a corresponding cypress.js smoke test. See the cypress/integration folder for examples.

The [comment by Peril](https://github.com/artsy/force/pull/8073#issuecomment-887261345) sounds actionable!

@anastasiapyzhik Would you be able to add a smoke test to the cypress/integration folder?

**Comment by anastasiapyzhik** [comment]
> > Routes added to routes.tsx should have a corresponding cypress.js smoke test. See the cypress/integration folder for examples.
> 
> The [comment by Peril](https://github.com/artsy/force/pull/8073#issuecomment-887261345) sounds actionable!
> 
> @anastasiapyzhik Would you be able to add a smoke test to the cypress/integration folder?

Added 🙂

File: cypress/integration/fairOrganizer.spec.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { visitWithStatusRetries } from "../helpers/visitWithStatusRetries"
+
+describe("FairOrganizer", () => {
+  it("/fair-organizer/:slug", () => {
+    visitWithStatusRetries("fair-organizer/expo-chicago-2019")
```
Comment (dblandin) [inline cypress/integration/fairOrganizer.spec.ts:5]: I don't think we're fetching any real data at the moment but I think the slug would be `fair-organizer/expo-chicago` for the organizer page. `expo-chicago-2019` refers to the specific fair slug.

**Review by mdole — APPROVED** [review]
Agreed with Devon's note about `expo-chicago-2019` -> `expo-chicago`, otherwise this looks great! Assigning back to you @anastasiapyzhik so you can merge once that's updated

**Pull request merged**